regex-assembly/942180.ra

```diff
@@ -3,27 +3,47 @@
 
 ##!+ i
 
-\d[\"'`]\s+[\"'`]\s+\d
-^admin\s*?[\"'`]
-(?:/\*)+[\"'`]+\s?
-(?:/\*)+[\"'`]+\s?--
-(?:/\*)+[\"'`]+\s?#
-(?:/\*)+[\"'`]+\s?/\*
-(?:/\*)+[\"'`]+\s?{
-[\"'`]\s*?or[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?xor[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?div[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?like[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?between[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?and[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?[^\w\s]?=\s*?[\"'`]
-[\"'`]\W*?[+=]+\W*?[\"'`]
-[\"'`]\s*?[!=|][\d\s!=+-]+.*?[\"'`(].*?$
-[\"'`]\s*?[!=|][\d\s!=]+.*?\d+$
-[\"'`]\s*?like\W+[\w\"'`(]
+
+##!> define open-comment (?:/\*)
+
+##!> define quotes [\"'`]
+##!> define quotes-plus-digits [\d\"'`]
+##!> define comment-suffix (?:--|#|\/*|\{)
+##!    PB: Should comment-suffix also include "\*/" ?
+
+##!> define logical-operators or|xor|and
+##!> define math-operators div
+##!> define comparison-operators like|between
+##!> define math-symbols [+<>=(),-]
+
+{{open-comment}}+{{quotes}}+\s?{{comment-suffix}}?
+
+##!> assemble
+  {{quotes}}\s*
+  ##!=>
+  {{logical-operators}}
+  {{math-operators}}
+  {{comparison-operators}}
+  ##!=>
+  [\w\s-]+{{math-symbols}}\s*{{quotes-plus-digits}}
+  ##!=>
+##!<
+
+\d{{quotes}}\s+{{quotes}}\s+\d
+
+{{quotes}}[<>~]+{{quotes}}
```

Comment:
You can put lines 34, 38, and 40 into a single assemble block with a `{{quotes}}` prefix and suffix.